src/test/java/org/apache/datasketches/theta/PairwiseSetOperationsTest.java

```diff
@@ -171,7 +171,7 @@ public void checkAnotBEarlyStop() {
       CompactSketch csk2 = usk2.compact(true, null);
 
       Sketch rsk = PairwiseSetOperations.aNotB(csk1, csk2);
-      double result1 = rsk.getEstimate();
+      double result1 = rsk.getEstimate(); //null ptr
```

Comment:
These were comments I was making in the code as I was troubleshooting and forgot to go back and clean up.  The entire PairwiseSetOperations class is now deprecated since the main set operation classes now cover these pair operations as part of their API.   Eventually, when we move to version 3.0, we can delete the deprecated classes and the tests associated with them.